### DISCUSSION TITLE
Postges Serial column type

### DISCUSSION THREAD

**Discussion by gmbronco** [ponder-sh/ponder#1270] [Question]
I’m curious why serial types were removed from the original Drizzle. Is there a Ponder-specific method for setting up integer auto-incrementing primary keys in the database? Thank you!

**Comment by kyscott18** [comment]
We don't have an auto incrementing column type yet. The reason being is that the store methods (`db.find()`, `db.insert` ...) run in-memory rather than as sql queries for performance reasons. There's nothing fundamental about not supporting them, just haven't implemented it yet. Right now most users use something like `log.id` as a unique key.
